build-dependents.js

```diff
@@ -23,7 +23,7 @@ try {
 
   if (isDependent || changedProject === currentProject) {
     // Rebuild the current project
-    const command = `nx run-many --target=d --projects=${currentProject} --parallel=5 --no-cloud`;
+    const command = `nx run-many --target=d --projects=${currentProject} --no-cloud`;
```

Comment:
why `no-cloud` for this? is it to skip caching hit?

Reply:
reduce parallel count for flakiness?

Reply:
no-cloud here will keep local cache, but skip the remote cache. With the dev mode being used mostly for constant changes, trying to hit the cloud again resulted in wasted resources and added some time to the dev rebuild loop. The no-cloud was added a bit ago to save the few seconds that were being wasted hitting the cloud at the start of each dev loop unnecessarily.

The parallel count was just moved to a common value in the nx.json rather than being hard coded everywhere, and was bumped to 8 to maximize the 8 core usage of our runners in the more workload heavy workflows 😄